tests/ApplicationTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Console\Tests;
+
+use Psr\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use Yiisoft\Yii\Console\ExitCode;
+use Yiisoft\Yii\Console\Event\ApplicationStartup;
+use Yiisoft\Yii\Console\Event\ApplicationShutdown;
+
+final class ApplicationTest extends TestCase
+{
+    public function testGetDispatcher(): void
+    {
+        $this->assertInstanceOf(EventDispatcherInterface::class, $this->application->getDispatcher());
+    }
+
+    public function testDispatcher(): void
+    {
+        $event = new ApplicationStartup();
+
+        $dispatcher = $this->application->getDispatcher();
+        $result = $dispatcher->dispatch($event);
+
+        $this->assertEquals($event, $result);
+
+        $event = new ApplicationShutdown(ExitCode::OK);
```

Comment:
These two tests should be in two test methods, not one.

Reply:
done.